### PR TITLE
Release v0.3.2

### PR DESCRIPTION
**Pull request by ganemone**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,8 +41,7 @@
     "lint": "eslint . --ignore-path .gitignore",
     "transpile": "npm run clean && cup build",
     "build-test": "rm -rf dist-tests && cup build-tests",
-    "just-test":
-      "node_modules/.bin/unitest --browser=dist-tests/browser.js --node=dist-tests/node.js",
+    "just-test": "node_modules/.bin/unitest --browser=dist-tests/browser.js --node=dist-tests/node.js",
```
Comment (nadiia) [inline package.json]: shouldn't be in a release pr?

**Comment by mlmorg** [comment]
What does this do?

**Comment by mlmorg** [comment]
Changelog?

**Comment by ganemone** [comment]
The changes should be fully internalized, as nothing that was changed was exposed to the user via an API. Also, afaik we are not manually writing the changelogs correct? Only migration guides if there is any migration needed

**Comment by ganemone** [comment]
!merge

**Pull request merged**